modules/developers-guide/pages/cql-using.adoc

```diff
@@ -0,0 +1,171 @@
+= Using the CQL API
+
+To use Stargate's Cassandra Query Language (CQL) API you can use the same CQL
+syntax, tools and native language drivers that you would with a normal
+Cassandra deployment.
+
+Popular Cassandra drivers are:
+
+* https://github.com/datastax/java-driver[Java]
+* https://github.com/datastax/csharp-driver[C#]
+* https://github.com/datastax/cpp-driver[C++]
+* https://github.com/datastax/python-driver[Python]
+* https://github.com/datastax/nodejs-driver[Node.js]
+* https://github.com/gocql/gocql[Go]
+* https://spring.io/projects/spring-data-cassandra[Spring Data Cassandra]
+
+[NOTE]
+====
+When using Cassandra drivers with Stargate you must:
+
+. Use the addresses of the Stargate instances as contact points rather than the
+addresses of the Cassandra storage instances.
+. Use the Round Robin policy in the driver for the
+https://docs.datastax.com/en/devapp/doc/devapp/driversLoadBalancing.html[Load Balancing Policy]
+rather than the Token Aware Policy.
+====
+
+include::quickstart:page$quick_start-rest.adoc[tag=prereqsList]
+
+* cqlsh (standalone, or as part of a separate Cassandra installation)
+
+include::quickstart:page$quick_start-rest.adoc[tag=getDockerImage]
+
+include::quickstart:page$quick_start-rest.adoc[tag=startDocker]
+
+=== Installing standalone cqlsh using Docker
+
+[source,bash]
+----
+docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' stargate
+docker pull nuvo/docker-cqlsh
+----
+
+// Use `pip` to install the standalone cqlsh tool.
+//
+// NOTE: cqlsh requires Python 2.7, and won't work with Python 3.
+//
+// [source,bash]
+// ----
+// pip install cqlsh
+// ----
+
+== Using the CQL API using cqlsh
+
+Run `cqlsh` from the Docker image.
+
+[source,bash]
+----
+docker run --rm -it nuvo/docker-cqlsh cqlsh localhost --cqlversion=3.4.5
```

Comment:
We should move this cmd down to this section and change it to:

Get the IP address of your Stargate docker container (note this example assumes that the Stargate container name is "stargate")
```
docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' stargate
```

Connect to your Stargate container using cqlsh
```
docker run --rm -it nuvo/docker-cqlsh cqlsh {ip-address} --cqlversion=3.4.5
```